src/wp-content/themes/twentytwentyfour/README.md

```diff
@@ -0,0 +1,209 @@
+# Twenty Twenty-Four
+
+Welcome to the development repository for the default theme that will launch with [WordPress 6.4](https://make.wordpress.org/core/6-4/).
+
+<img width="1920" alt="2023" src="https://github.com/WordPress/twentytwentyfour/assets/1813435/d965b75d-03cd-4365-b963-b3862d482329">
```

Comment:
```suggestion
<img width="1920" alt="2024" src="https://github.com/WordPress/twentytwentyfour/assets/1813435/d965b75d-03cd-4365-b963-b3862d482329">
```

Reply:
I'm thinking maybe this README file shouldn't go in core, since it's specific to development of the theme. But I will make the change on the theme repo